devex/vsextension/msbuild/edl-t.targets

```diff
@@ -5,11 +5,11 @@
   <ItemGroup>
     <PropertyPageSchema Include="$(MSBuildThisFileDirectory)edl.xml" />
     <AvailableItemName Include="EdlItem">
-      <Targets>_OEGenerateTrustedCode</Targets>
+      <Targets>_OEUtilTrustedCode</Targets>
```

Comment:
This is for generating trusted code. Unrelated to `oegenerate` tool. Please revert.